Added react-responsive library to determine if it is mobile screen or …

AdnanLatif opened the pull request:
…desktop

- Used react-responsive library to get device width and check if it is mobile than show text "Join now" if not mobile device than show text "Become a member"

AdnanLatif committed:
Added react-responsive library to determin if it is mobile screen or desktop

- Used react-responsive library to get device width and check if it is mobile than show text "Join now" if not mobile device than show text "Become a member"

tauqeernasir commented on src/screens/home/components/members-list.tsx:
```diff
@@ -45,7 +49,10 @@ export function MembersList(props: MembersListProps) {
         <Link href='/join'>
-          <a><Text as='span' color='gray.400' ml='20px'>Become a member</Text></a>
+          <a><Text as='span' color='gray.400' ml='20px'>
+            { isDesktopOrLaptop && <>Become a member</> }
+            { !isDesktopOrLaptop && <>Join now</> }
```
```suggestion
            { isDesktopOrLaptop ? 'Become a member' :  'Join now' }
```

AdnanLatif committed:
Update src/screens/home/components/members-list.tsx

Suggest changes by tauqirnasir

Co-authored-by: Tauqeer Nasir <60596259+tauqeernasir@users.noreply.github.com>

tauqeernasir reviewed (approved):
Thank you, LGTM.

AdnanLatif commented:
Thank you for approving my commit.

nilbuild commented on .gitignore:
```diff
@@ -31,4 +31,7 @@ yarn-error.log*
 
 # vercel
-.vercel
+.
+
+# package-json.lock
+package-json.lock
```
Let's not ignore it

AdnanLatif replied:
Removed package-json.lock from .gitignore

nilbuild commented on package.json:
```diff
@@ -22,6 +22,7 @@
     "react": "17.0.1",
     "react-dom": "17.0.1",
     "react-progressive-image": "^0.6.0",
+    "react-responsive": "^8.1.0",
```
No need for this, chakra-ui has the responsive utilities by default

AdnanLatif replied:
Removed react-responsive.

nilbuild commented on src/screens/home/components/members-list.tsx:
```diff
@@ -45,7 +49,9 @@ export function MembersList(props: MembersListProps) {
         </Button>
         <Link href='/join'>
-          <a><Text as='span' color='gray.400' ml='20px'>Become a member</Text></a>
+          <a><Text as='span' color='gray.400' ml='20px'>
+            { isDesktopOrLaptop ? 'Become a member' :  'Join now' }
+            </Text></a>
```
No need for react-responsive, we can just do
```suggestion
          <a>
            <Text d={['none', 'none', 'inline', 'inline']} as='span' color='gray.400' ml='20px'>Become a member</Text>
            <Text d={['inline', 'inline', 'none', 'none']} as='span' color='gray.400' ml='20px'>Join now</Text>
          </a>
```

AdnanLatif replied:
Changes done as per your instructions.

AdnanLatif committed:
Removed package-json.lock from .gitignore

AdnanLatif committed:
Changes in member-list file as per Kamran's suggestion

- Removed react-responsive library and used chakra ui built-in responsive method.

AdnanLatif commented:
Changes in member-list file as per Kamran's suggestion

nilbuild reviewed (approved):
Thanks
